sb-cb-ui-utils-v2/library/sunbird-cb/utils-v2/src/lib/services/configurations.model.ts

```diff
@@ -68,6 +68,7 @@ export namespace NsInstanceConfig {
     websitelanguages: any[]
     profileTimelyNudges: any,
     npsCategory?: string
+    compentency: any
```

Comment:
it should be of specific type either string , instead of any

Reply:
It doesn't have any specific type it's not a string, it's an object and can have any parameter in it